### ISSUE TITLE
v1.2.1 doesn't find JUnit5 Instrumented Tests

### ISSUE THREAD

**Issue by sumio** [mannodermaus/android-junit5#241]
JUnit5 tests don't run if I run Instrumented Tests (androidTest) by using Instrumentation v1.2.1.
Only JUnit4 tests run.

### Reproduced Version

- Instrumentation: 1.2.1
- Plugin: 1.7.1.1

### Steps to reproduce

I made the reproducing project that leaves only your `instrumentation/sample` directory in https://github.com/sumio/android-junit5/tree/junit5-test-not-found/instrumentation

1. Launch an Android Emulator
2. Run the following commands:  
    ```sh
    git clone git@github.com:sumio/android-junit5.git -b junit5-test-not-found
    cd android-junit5/instrumentation/
    ./gradlew clean connectedDebugAndroidTest
    open sample/build/reports/androidTests/connected/flavors/debugAndroidTest/index.html
    ```
3. You can see only `TestRunningOnJUnit4` run

If you rewrite to `1.2.0` the following line
https://github.com/sumio/android-junit5/blob/junit5-test-not-found/instrumentation/sample/build.gradle#L34 and
do the same steps again, not only `TestRunningOnJUnit4` but also `ActivityOneTest` and `TestRunningOnJUnit5` will be executed correctly.

**Comment by mannodermaus** [comment]
Thanks for reporting! I was under the impression that nothing changed between 1.2.0 and 1.2.1 in terms of the library except for the transition to Maven Central. I'm suspecting some weird changes in the transitive dependencies embedded in the POM or something like that. I'll have time shortly to look into this.

**Comment by mannodermaus** [comment]
I have identified the problem to be indeed related to missing runtime-only dependencies in the POM file, caused by the transition away from the Bintray plugin. Will create a new version momentarily with the full list, sorry abou that.

As a workaround for right now, either downgrade to 1.2.0 or add the mssing transitive deps manually:

```kotlin
dependencies {
    // ...
    androidTestRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:5.7.1"
    androidTestRuntimeOnly "org.junit.platform:junit-platform-runner:1.7.1"
}
```

**Comment by mannodermaus** [comment]
Resolved in 1.2.2. Thanks again for reporting. 🙏

**Comment by sumio** [comment]
I also confirmed that the 1.2.2 ran successfully all tests including JUnit5 in our project.
Thank you very much!
